README.md

```diff
@@ -233,6 +233,23 @@ name.valueOf();
 // ➜ { block: 'button', mod: { name: 'focused', value: true } }
 ```
 
+### isBemEntityName()
+
+Determines whether specified entity is instance of BemEntityName.
+
+Parameter    | Type            | Description
+-------------|-----------------|-----------------------
+`entityName` | `BemEntityName` | The entity to check.
+
+```js
+const BemEntityName = require('@bem/entity-name');
+
+const entityName = new BemEntityName({ block: 'input' });
+
+BemEntityName.isBemEntityName(entityName); // true
```

Comment:
see https://github.com/bem-sdk/bem-entity-name/pull/2

`instanceof` will not work with different versions of `BemEntityName`.